apps/end-tenancy/index.js

```diff
@@ -33,6 +33,9 @@ module.exports = {
         'report-link': 'https://eforms.homeoffice.gov.uk/outreach/lcs-reporting.ofml'
       }
     },
+    '/contact': {
+      controller: require('./controllers/content')
```

Comment:
I'll update, as yes this is a good point. When node finally move to the ES6 import style, this shit is all hoisted anyway